templates/macros.xml.j2

```diff
@@ -2,38 +2,9 @@
 <!-- {{ ansible_managed }} -->
 <yandex>
     <macros>
-        <cluster>{{ clickhouse_display_name | default('clickhouse') }}</cluster>
-        {% if inventory_hostname in groups['clickhouse_shard01'] %}
-        <shard>01</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard02'] %}
-        <shard>02</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard03'] %}
-        <shard>03</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard04'] %}
-        <shard>04</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard05'] %}
-        <shard>05</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard06'] %}
-        <shard>06</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard07'] %}
-        <shard>07</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard08'] %}
-        <shard>08</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard09'] %}
-        <shard>09</shard>
-        {% endif %}
-        {% if inventory_hostname in groups['clickhouse_shard10'] %}
-        <shard>10</shard>
-        {% endif %}
+        <cluster>{{ cluster }}</cluster>
+        <shard>{{ shard }}</shard>
         <replica>{{ inventory_hostname }}</replica>
-        <distributed_clusters>distributed_cluster</distributed_clusters>
+        <distributed_clusters>{{ distributed_cluster }}</distributed_clusters>
```

Comment:
was it working before and now we want to make it dynamic?

Reply:
yes correct